seqr/migrations/0047_alter_project_is_mme_enabled.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.15 on 2022-10-14 03:07
```

Comment:
Did you include this deliberately? How is this related to the PR?

Reply:
Ah yes I did, this should have been included in the merge - I checked and the schema appliers should not be makingmigrations, these should be included and they should just be applied.